### ISSUE TITLE
The network detects 18 different (farther than 0.99) me when moving my head.

### ISSUE THREAD

**Issue by educob** [ZhaoJ9014/face.evoLVe.PyTorch#27]
Hi.

I have developed a little app that when detecting a face tries to find it in the database and if it's not there, it created a new record.
As I move my head the app thinks it's a new user (distance >= 0.99) and creates a new entry in the database.
Up to 18 records have been created.

I was thinking of developing a professional app for controlling access to places, etc. 
How can I do to filter (or any other way) these extra records created with different head poses?

Thanks.

**Comment by justein** [comment]
@educob have you retrained the model？

**Comment by educob** [comment]
No. I was told in other issue days ago I didn't need to train it.

The app would be incorporating new people every day so I don't know the kind of training I should have to do. I am quite disappointed cause the idea I had in mind was quite powerful.
